README.md

```diff
@@ -1,24 +1,194 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-* Ruby version
+  <h3><b>Microverse README Template</b></h3>
 
-* System dependencies
+</div>
+<!-- TABLE OF CONTENTS -->
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database initialization
+<!-- PROJECT DESCRIPTION -->
 
-* How to run the test suite
+# 📖 Doctor Appointment Application <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+> A mobile web application that allows for the booking of appointment from a list of available doctors with different specialties. It displays list of available doctors, logged in users can book an appointment, delete and modify appointment bookings.
 
-* Deployment instructions
+## Frontend Repo 
+- [doctor_appointment_frontend](https://github.com/ogagaoghene/doctor_appointment_frontend.git)
 
-* ...
+## Kanban Board
+
+We have set up a Kanban board
+note: We are 2 students working on this project.
+
+Kanban board link:
+https://github.com/users/ogagaoghene/projects/13
```

Comment:
[OPTIONAL] Consider a live deployment section that shows working example of deployment (if there are swagger docs associated that will be helpful to user understanding how the application works before decing if they'd like to clone it or not)

Reply:
@mcrd25 Many thanks to you for this optional suggestion. I will add the API documentation to this. But, Heroku is free anymore with a PostgreSQL database. I would have love to do this but I would prefer to include the API documentation for this.